docs/Makefile

```diff
@@ -52,6 +52,7 @@ clean:
 	rm -rf $(BUILDDIR)/*
 
 html:
+	git config --global alias.co checkout
```

Comment:
You absolutely do not want `--global` (writting to `~/.gitconfig`) because this impact the user's wide config.  Like for file formatting, keep it for the specific project only, not user's wide.

But can we just take a step back and understand why you need to add an alias in the first place?  If some script use `git co`, why not just modify the script instead to spell out the full `git checkout`.  Git alias is meant to save typing for the human users.  We do not need to save typing for a script.

Fooling around with git alias is usually a bad idea anyways.  What if the user already map `git co` to something else and now you're overriding his custom config?

Reply:
@tlvu It was not compiling without `--global`. 

Understand all the rest... Any better solution? 
Gues the `git co` alias is coming form the sphinx build, so it might be possible to get rid of the alias there (but don't know where exactly the `co` is being set)

Whats your suggestion?

Reply:
use `--local`

Reply:
> Gues the `git co` alias is coming form the sphinx build / `gitinclude`, so it might be possible to get rid of the alias there (but don't know where exactly the `co` is being set)
> 

Could you try fixing this line https://github.com/bird-house/birdhouse-docs/blob/7b36c72948c67b0509e6ca3b1b9c0c79af7a6959/docs/source/ext/docaggregation.py#L20 make it `git checkout` instead.

Reply:
@tlvu That's the trick!